setup.py

```diff
@@ -111,6 +111,8 @@ def build_extension(self, ext: CMakeExtension) -> None:
         extdir = ext_fullpath.parent.resolve()
 
         output_dir = f"{extdir}{os.sep}"
+        gpp_path = subprocess.run(['which', 'g++'], capture_output=True).stdout.decode('ascii').replace('\n', '')
```

Comment:
can you check whether it is on linux since ```which``` command may not work on windows

Reply:
@park12sj Are you going to take care of the support on windows?

Reply:
Yes I think this will break maintains of other compiles and overwrite user input via `os.environ.get("CMAKE_ARGS", "")`. In addition, `stdout.decode('ascii')` sounds problematic if there are some CJK characters in the path.

Regarding your situations in the #743, you will get different CMAKE_CXX_COMPILER / CMAKE_C_COMPILER anyway and you have to make them >= version 8. In addition, it may help to remove the `build` directory if you have tried to build ITREX before with another compiler version (e.g. run the build command in a different conda env with different version of gcc).

Reply:
@airMeng
I watched the moment late. I'm sorry.
I understood that you're going to revert in the pr below, is that correct?
https://github.com/intel/intel-extension-for-transformers/pull/790

Reply:
@DDEle 

In relation to this, even if the gcc version is updated, if it is not specified as the corresponding pr, the build will be executed with the existing gcc when executing `pip install . `
The same is true if erase the build directory or run it on another conda env.
This seems to be my environmental issue rather than the issue of itrex, so I'll look for another way.

```
scl enable devtoolset-11 zsh
pip install -v .

...

  CMake Error in CMakeLists.txt:
    The compiler feature "cxx_std_17" is not known to CXX compiler

    "GNU"

    version 4.8.5.
```

Reply:
Interesting... In my experience, cmake always find the current gcc/g++ in the environment.

Maybe you can try clearing the build dir and running`CC=$(which gcc) GXX=$(which g++) pip install -v .`?